device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -36,17 +33,16 @@
  * </p>
  * The client supports HTTPS 1.1 and AMQPS 1.0 transports.
  */
-@SuppressWarnings("deprecation") // A lot of references to deprecated APIs, but all of them are still in use
 @Slf4j
 public final class DeviceClient extends InternalClient implements Closeable
 {
-    private static long SEND_PERIOD_MILLIS = 10L;
+    private static final long SEND_PERIOD_MILLIS = 10L;
 
-    private static long RECEIVE_PERIOD_MILLIS_AMQPS = 10L;
-    private static long RECEIVE_PERIOD_MILLIS_MQTT = 10L;
-    private static long RECEIVE_PERIOD_MILLIS_HTTPS = 25*60*1000; /*25 minutes*/
+    private static final long RECEIVE_PERIOD_MILLIS_AMQPS = 10L;
+    private static final long RECEIVE_PERIOD_MILLIS_MQTT = 10L;
+    private static final long RECEIVE_PERIOD_MILLIS_HTTPS = 25*60*1000; /*25 minutes*/
 
-    private IoTHubConnectionType ioTHubConnectionType = IoTHubConnectionType.UNKNOWN;
```

Comment:
Calling this a "IoTHubConnectionType" was a bit misleading since it really is only used to differentiate between multiplexed device clients a non-multiplexed device clients. It's an enum that is only used internally though so no user will care.